core/__tests__/models/appRefreshQuery.ts

```diff
@@ -78,6 +77,28 @@ describe("appRefreshQuery", () => {
       appRefreshQuery.destroy();
     });
 
+    test("an updated query on a ready appRefreshQuery enqueues a run", async () => {
+      await api.resque.queue.connection.redis.flushdb();
+
+      const appRefreshQuery = new AppRefreshQuery({
+        appId: app.id,
+        refreshQuery: "SELECT MAX(updated_at) FROM users;",
+        state: "ready",
+      });
+      await appRefreshQuery.save();
+
```

Comment:
I think here there should be check that there wasn't a task enqueued the first time when the  AppRefreshQuery was created.

Reply:
I'm not sure why we wouldn't want it to enqueue then if the state is ready and there's a query.

But I do think we should flush redis before the update in that case.